R/utils.R

```diff
@@ -567,3 +567,26 @@ complete_observations <- function(data, id, within, dv) {
 
 }
 
+
+#' Add Equals Where Necessary
+#'
+#' This is an internal function that prepends every element of a character
+#' vector with an 'equals' sign if the respective element does not contain one of
+#' \code{c("=", "<", ">")}.
+#'
+#' @param x A character vector.
+#'
+#' @keywords internal
+
+
+add_equals <-function(x) {
```

Comment:
Nice. Would you be willing to search the code base for other instances where this could be used?

Reply:
Sure.